### ISSUE TITLE
Is there an option like blastp to set word size for short sequences?

### ISSUE THREAD

**Issue by graa4** [bbuchfink/diamond#802]
Hi, I would like to know if the length of the amino acid sequence is only 40 to 50, how to set parameters such as word size like blastp for such a sequence?

**Comment by bbuchfink** [comment]
No, this corresponds to different sensitivity modes of diamond.
